dir.props

```diff
@@ -36,6 +36,7 @@
     <BaseOutputPath>$(ProjectDir)bin/</BaseOutputPath>
     <ToolsDir>$(ProjectDir)Tools/</ToolsDir>
     <TaskDirectory>$(ProjectDir)tools-local/tasks/</TaskDirectory>
+    <TargetDirectory>$(ProjectDir)tools-local/targets/</TargetDirectory>
```

Comment:
Is there an advantage to having this new, separate directory for just one file?  Could we put the `update-submodules.targets` file in the root?

Reply:
That would be fine. I do consider it a local tool, though, and I figured the fewer things in the root, the better (usually).

Reply:
I wouldn't put it in the root nor in its own directory I would just put it in the tools-local folder directly.

Reply:
Will do.